Docker/README.md

```diff
@@ -74,27 +74,35 @@ After initial `docker run`,
 * Build the Docker image from `Dockerfile` definition
 * Test image
 * Integrate with CI (e.g. Travis CI) [Example of CI in docker images](https://github.com/docker-library/ruby/blob/master/.travis.yml)
-* Push to [Docker Hub](#updating-image-on-docker-hub)
+* Configure [Docker Hub](hub.docker.com) to [autobuild your image](#autobuilding-image-on-docker-hub)
 * Test image from Docker Hub
-* If applicable, test Docker Compose 
+* If applicable, test Docker Compose
 
-<a name="updating-image-on-docker-hub"></a>
-### Updating image on Docker Hub 
+<a name="autobuilding-image-on-docker-hub"></a>
+### Autobuilding image on Docker Hub
 
-* Remove local container, if present
```

Comment:
Should we keep the "Updating image on Docker Hub" section, move it down below the autobuild section, just in cases people need to do manual provision in some rare cases?